docs/UserGuide.md

```diff
@@ -228,44 +296,130 @@ Clears all tasks in the list.
 Format: `cleartask`
 
 
-### Contact Features
+### Contact List Features
+
 
-#### Add a contact: `addcontact`
+#### Adding a contact: `addcontact`
 
 Adds a new contact into the contact list if it does not already exist.
 
-Format: `addcontact n/NAME [e/EMAIL] [te/TELEGRAM] [t/TAG]`
+Format: `addcontact n/NAME e/EMAIL [te/TELEGRAM] [t/TAG]...`
+   
+ * The `TELEGRAM` field provided must start with the `@` symbol and must be a valid telegram username,
+   i.e. at least 5 characters long, not including the `@` symbol and contains only alphanumeric characters or underscore
+ 
+ * A contact can have any number of tags (including 0)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** 
-Users are required to provide at least one other contact detail apart from **Name**.
-</div>
 
-Examples of Usage:
-`addcontact n/john e/john@gmail.com te/@john123`
+Examples:
+
+ * `addcontact n/john e/john@gmail.com`
+ * `addcontact n/amy e/amy@gmail.com te/@anytele`
+ * `addcontact n/bob e/bob@gmail.com t/friend`
+ 
+ 
+ #### Listing all contacts: `listcontact`
+ 
+ Shows a list of all contacts in the contact list.
+ 
+ Format: `listcontact`
```

Comment:
Maybe we can add an additional tip to the user where `listcontact` is useful to reset to the original list after a `findcontact` or `sortcontact` commands?

This is also one of the requirements for CS2101 to add tips for the user 😄 i.e. :

"[Taken from UG Info Pack] 5. Highly considerate of user needs by providing other relevant and helpful information (e.g. warnings, tips, important note)"

Reply:
Yes, I will add this in. Thanks for the note 👍